server/src/main/java/org/opensearch/index/remote/RemoteSegmentStats.java

```diff
@@ -231,6 +262,8 @@ public void writeTo(StreamOutput out) throws IOException {
          */
```

Comment:
There are some comments that can be removed now or since we are adding couple of more fields, we can remove them when backport the final change with the exact version.

Reply:
Marking the version check with `Version.CURRENT` to ensure BwC. Will remove this compatibility check once the code is backported to 2.x